examples/autoinstrument/README.md

```diff
@@ -8,8 +8,8 @@ export GOOGLE_CLOUD_PROJECT={your-project}
 
 ./gradlew :examples-autoinstrument:jib --image="gcr.io/$GOOGLE_CLOUD_PROJECT/hello-autoinstrument-java"
 
-kubectl create deployment hello-autoinstrument-java \
-  --image=gcr.io/$GOOGLE_CLOUD_PROJECT/hello-autoinstrument-java
+sed s/%GOOGLE_CLOUD_PROJECT%/$GOOGLE_CLOUD_PROJECT/g \
```

Comment:
Optional: consider using `envsubst` rather than `sed`.

Reply:
`envsubst` does seem simpler than using `sed`. A potential downside is that it is not pre-installed on some systems like MacOS - whereas `sed` is.